Getting-Started-with-Akka-http-signature.md

```diff
@@ -18,10 +18,10 @@ One can then import it into their scripts from inside the ammonite shell. The us
 ## The Key Pair
 
 The Akka-Http-Signature library uses asymmetrical cryptography. As detailed in the [Public-Key cryptography wikipedia page](https://en.wikipedia.org/wiki/Public-key_cryptography), this is a cryptographic system that uses pairs of private and public keys. The library uses them to form Digital Signatures. In such a system public keys can be looked up by anyone and private keys are only known by their owner. Communication is achieved the following way : 
-1. The sender projects a message signed by his own private key 
+1. The sender sends a message with a header that is signed by his own private key 
 2. The message can only be verified by clients that have access to that sender's public key
 
-Because the messages are signed by the sender's private key and can only be verified via that sender's public key - only the intended clients (one that have access to said key) can ever receive the information within the message. This also ensures that the message is not changed in any way as the signature is bound to the original message and verification will fail if any alterations have occured.
+Because the messages have headers that are signed by the sender's private key and can only be verified via that sender's public key - only the intended clients (one that have access to said key) can ever receive the information within the message. With the right signature, this cryptographic system can also ensure that the message is not changed in any way as the signature is bound to the original message and verification will fail if any alterations have occured.
```

Comment:
> only the intended clients (one that have access to said key) can ever receive the information within the message.

actually that is either true and an odd way of putting it, or false. 
It is true because  public key being public *anyone* can have access to it. 
(It is true in in an  HTTP message with a signature is sent over TLS, but that is due to TLS encryption of the channel )

So the question to you is: what headers have to be signed for there to be a garantee that the message was not altered?